_RFCs/0001-rfc-for-rfcs.md

```diff
@@ -0,0 +1,185 @@
+
+Furthermore, the fact that a given RFC has been accepted and is "active" implies nothing about what priority is assigned to its implementation, nor does it imply anything about whether a Core-Plans developer has been assigned the task of implementing the change. While it is not *necessary* that the author of the RFC also write the implementation, it is by far the most effective way to see an RFC through to completion: authors should not expect that other project developers will take on responsibility for implementing their accepted feature.
+
+Modifications to "active" RFCs can be done in follow-up pull requests. We strive to write each RFC in a manner that it will reflect the final design of the feature; but the nature of the process means that we cannot expect every merged RFC to actually reflect what the end result will be.
+
+In general, once accepted, RFCs should not be substantially changed. Only very minor changes should be submitted as amendments. More substantial changes should be new RFCs, with a note added to the original RFC. Exactly what counts as a "very minor change" is up to the core-plans maintainers to decide.
+
+## Reviewing RFCs
+[Reviewing RFCs]: #reviewing-rfcs
+
+When the RFC pull request is first opened, a core plans maintainer will be assigned as a shepherd to the RFC to ensure it makes as much progress through the RFC pipeline as is possible. While the RFC PR is open the assigned shepherd may schedule meetings  with the author and/or relevant stakeholders to discuss the issues in greater detail, and in some cases the topic may be discussed at a maintainers meeting. In either case a summary from the meeting will be posted back to the RFC pull request.
+
+Maintainers make the final decisions about RFCs after the benefits and drawbacks are well understood. These decisions can be made at any time, but the maintainers will regularly issue decisions. When a decision is made, the RFC pull request will either be merged or closed. In either case, if the reasoning is not clear from the discussion in thread, the maintainers will add a comment describing the rationale for the decision.
+
+
+## Implementing an RFC
+[Implementing an RFC]: #implementing-an-rfc
+
+Some accepted RFCs represent vital changes that need to be implemented right away. Other accepted RFCs can represent changes that can wait until some arbitrary contributor feels like doing the work. Every accepted RFC has an associated issue tracking its implementation in the Core-Plans repository; thus that associated issue can be assigned a priority via the triage process that the team uses for all issues in the Core-Plans repository.
+
+The author of an RFC is not obligated to implement it. Of course, the RFC author (like any other contributor) is welcome to post an implementation for review after the RFC has been accepted.
```

Comment:
We should try to avoid unfunded mandates though.  This as an early problem with the Chef RFC process.

Reply:
Oh weird I thought I had something in here on that. It might just be in a separate section? Looking now, but if I cant find it ill add some clarification to this.